spec/lib/staff_constraint_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe StaffConstraint do
+  subject { described_class.new }
+
+  describe '#matches?' do
+    context 'when without user session' do
+      let(:request) { double('request', session: {}) }
+
+      it { expect(subject.matches?(request)).to be_falsey }
```

Comment:
Just noticed this was already merged. If you're feeling extra nice, bonus :sparkles: for a follow up PR.